### ISSUE TITLE
Mention CTRL + ALT + Arrow in the Orbiter manual

### ISSUE THREAD

**Issue by Urwumpe** [orbitersim/orbiter#349]
### Problem

The keybinding for leaning over in a VC is mentioned in the API reference, but not it the user manual. 

### Desired state

This should be included in the keybinding section for virtual cockpits

**Comment by schnepe2** [comment]
Looks like it is mentioned already in Orbiter.odt
![grafik](https://github.com/orbitersim/orbiter/assets/18400313/60c8446e-af33-4c88-b48a-4c040169c583)

...although my LibreOffice lacks a font it seems 😉 ("keystroke").
Anyone knows what font Martin used there?

**Comment by schnepe2** [comment]
I found a font that seems to fit, although I don't like the source where I got it from 🙈 🙉 🙊 - I would prefer a more official source.

![grafik](https://github.com/orbitersim/orbiter/assets/18400313/d8044802-67de-4302-bc28-7ec09a102a33)
Microsoft has a web-page listing that font[*], but I have not found a way to install it 😞 

[*] https://learn.microsoft.com/en-us/typography/font-list/keystrokes-mt

**Comment by dimitry-ishenko** [comment]
Would this work? https://www.fontspace.com/212-keyboard-font-f34592

> Totally free for Personal and Small Commercial Use. Please DO NOT resell!

**Comment by schnepe2** [comment]
Just took a quick look at it and it seems it lacks the arrow-keys ... which are **a must** for our purpose.
Let's see if I find "the original" on some old "Micosoft Development Kit" CDs from the 2000s  😁

**Comment by dimitry-ishenko** [comment]
> Just took a quick look at it and it seems it lacks the arrow-keys ... which are **a must** for our purpose.

qrst

![q r s t](https://see.fontimg.com/api/renderfont4/lmRZ/eyJyIjoiZnMiLCJoIjoxNjMsInciOjI1MDAsImZzIjo2NSwiZmdjIjoiIzAwMDAwMCIsImJnYyI6IiNGRkZGRkYiLCJ0IjoxfQ/cXJzdA/212-keyboard.png)

**Comment by schnepe2** [comment]
Ah, OK. Could you check if the "mapping" in `Orbiter.odt` fits?
I mean if, for example, page 62 shows the correct keys? I'm currently not on a machine with LibreOffice installed.

**Comment by dimitry-ishenko** [comment]
No, the mappings are different. I am on Linux and quite a few things in the document look wrong to me. They all might be part of the same font though.

I don't know what the licensing is on the Microsoft font, but it might be worth embedding one of the fonts into the document to avoid this problem.

**Comment by n7275** [comment]
Wouldn't it be better to replace the fonts with some images of the keys?

**Comment by dimitry-ishenko** [comment]
> Wouldn't it be better to replace the fonts with some images of the keys?

It would be a royal PITA. They are spread throughout the doc.

**Comment by dimitry-ishenko** [comment]
Could someone on Windows try the attached docs? One of them has the font embedded.

[doc1.odt](https://github.com/orbitersim/orbiter/files/14017115/doc1.odt)
[doc2.odt](https://github.com/orbitersim/orbiter/files/14017116/doc2.odt)

**Comment by dimitry-ishenko** [comment]
It works for me on Linux:

![Screenshot from 2024-01-22 20-22-47](https://github.com/orbitersim/orbiter/assets/2858291/e8d6574f-e713-4e27-87c7-93e20c338968)

**Comment by schnepe2** [comment]
> Could someone on Windows try the attached docs? _doc1.odt_ has the font embedded. _doc2.odt_ - doesn't.
> [...]
> NB: Open them one at a time, because if you have the doc with the embedded font open, it will propagate to the other doc. Or, open _doc2.odt_ first.

Same on Windows: _doc1.odt_ has the font embedded (a.k.a. "works"). _doc2.odt_ - doesn't (a.k.a. "works not").

I have to investigate a bit, but embedding "the orginal" Microsoft Font seems to be legal, so I might embedd that font.

**Comment by schnepe2** [comment]
Wow, embedding the font(s) - it seems there is no way to just embedd a specific one -
would increases the size quite a bit
![grafik](https://github.com/orbitersim/orbiter/assets/18400313/536ba410-59b0-4165-a70d-1b0e6bd37368)

**Comment by schnepe2** [comment]
After sving it with "Only embed fonts that are used in documents" option it **even** shrunk in size 😁...
![grafik](https://github.com/orbitersim/orbiter/assets/18400313/6390022a-ee63-457d-bd7f-bf3a0ca100a7)

I might commit this into a branch, so others (you) can check if that really worked.

**Comment by dimitry-ishenko** [comment]
> I might commit this into a branch, so others (you) can check if that really worked.

You can attach it here.

**Comment by schnepe2** [comment]
Sure I can...

[Orbiter.odt](https://github.com/orbitersim/orbiter/files/14027496/Orbiter.odt)

**Comment by schnepe2** [comment]
Grrrr when I unistall the font on my machine, the "new document" is rendered bad again...

But I have 3 permutations left 😉:

![grafik](https://github.com/orbitersim/orbiter/assets/18400313/48c8659a-7c45-48cb-b267-da4bac2441c0)

I'll report back ....

**Comment by schnepe2** [comment]
Strange....
I've installed the font, then saved the document with all combinations of "embedding" available (Doc00.odt, Doc01.odt, Doc10.odt & Doc11.odt).
Then I unistalled the font from my system and none of those were any good 😞 
Confusing...

**Comment by dimitry-ishenko** [comment]
Your doc doesn't work for me either. Here is what I had in _doc1.odt_:

![Screenshot from 2024-01-23 16-52-07](https://github.com/orbitersim/orbiter/assets/2858291/b55868b5-e96d-48e7-9e58-4c3af0df581c)

**Comment by dimitry-ishenko** [comment]
@schnepe2 if you unzip an ODT file, you can see if it has any fonts embedded in it. It will have the `Fonts` directory inside.

**Comment by dimitry-ishenko** [comment]
I don't know where to get the Keystroke font, but I was able to embed the free replacement I've found (linked above) into _Orbiter.pdf_ ([link](https://file.io/WKOjIWn9Bm8f)). I've only changed page 62:

![Screenshot from 2024-01-23 17-44-47](https://github.com/orbitersim/orbiter/assets/2858291/4dd2e365-6711-45e9-af69-6de13a4be457)

It would be better if someone found the original font and embedded that, as there are quite a few changes that will have to be made.

**Comment by schnepe2** [comment]
> [...] if you unzip an ODT file, you can see if it has any fonts embedded in it. It will have the `Fonts` directory inside.

Thanks for the tip!

The two files **with** embedded fonts both lack the one we'd like to have (`Key.ttf`) :

![grafik](https://github.com/orbitersim/orbiter/assets/18400313/21ecf6a4-5003-450a-a3d5-0ccd2cc20fae)

Don't know why that is, but we'll find out eventually 😀 



> It would be better if someone found the original font and embedded that, as there are quite a few changes that will have to be made.

Yes, finding them and replacing them would be the last option I would chose.

**Comment by dimitry-ishenko** [comment]
@schnepe2 where did you find `Key.ttf`? I can try if it works for me.

**Comment by schnepe2** [comment]
Here: https://www.wfonts.com/search?kwd=keystroke (But as I said I'm not sure whether that is a "reputable" source 😉 )

Another observation: I just opened a file with embedded fonst again and saved as a new file again (with the same settings), but now some other two fonts were included...it looks like it's a hit and miss 😞 

"Installing" the font under `C:\Windows\Fonts\` (instead of the usual `C:\Users\<user>\AppData\Local\Microsoft\Windows\Fonts\`) makes no difference. I just wanted to try that.

**Comment by dimitry-ishenko** [comment]
It doesn't embed for me either. It may have something to do with the fact that it only has glyphs in the private area... 🤷🏻‍♂️

**Comment by schnepe2** [comment]
It looks like Keystrokes TM font must be bought from somewere like here:
https://www.fonts.com/en/font/monotype/keystrokes
Those purchases allow for a "[Electronic Doc license](https://www.fonts.com/font/monotype/keystrokes/licenses)".
Possibly Martin had this font.

Possibly the "inofficial Keystroke" (that from my last post - we can't embed) is just not marked as "emeddable"?

**Comment by EvESpirit** [comment]
This issue can probably be closed, since the manual contains at least two references to this feature, and the way the project is set up right now allows doxygen to show the keystroke icons correctly in the compiled PDF.

<img width="705" height="294" alt="Image" src="https://github.com/user-attachments/assets/3abc6448-77f9-4105-951c-79dfc6259080" />

**Comment by schnepe2** [comment]
> This issue can probably be closed[...]

I 100% agree
